python/pylibcudf/pylibcudf/tests/test_nvtext_normalize.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+import pyarrow as pa
+import pylibcudf as plc
+import pytest
+from utils import assert_column_eq
+
+
+@pytest.fixture(scope="module")
+def input_data():
+    arr1 = ["éâîô\teaio", "ĂĆĖÑÜ", "ACENU", "$24.08", "[a,bb]"]
+    arr2 = ["a b", "  c  d\n", "e \t f "]
+    return pa.array(arr1), pa.array(arr2)
+
+
+def test_normalize_spaces(input_data):
+    _, input_col = input_data
```

Comment:
This is a weird pattern to have two data inputs and only use one of them. Can we make two fixtures instead?